src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js

```diff
@@ -35,6 +35,16 @@ function dismissUnsavedChangesModal() {
     .click();
 }
 
+function cancelChangesViaModal() {
+  cy.axeCheck();
+  cy.get('va-modal')
+    .shadow()
```

Comment:
neat! so this is a way to check into the custom element's shadow dom?

Reply:
Yep, cypress allows you to drill into the shadow dom with this and then even query within that. It lives behind a cypress config flag though AFAIK: https://github.com/department-of-veterans-affairs/vets-website/blob/80c7844052f4a849aa60d6816c27fb92ba07501c/config/cypress.config.js#L195 otherwise I dont think this `shadow()` would work

https://docs.cypress.io/guides/references/configuration#Global